docs/contents/ions.rst

```diff
@@ -0,0 +1,109 @@
+Ions
+====
+
+NEURON supports computing the ion currents. For each segment there can be
```

Comment:
...computing the ion currents and ion concentrations.

Reply:
added.